test/controllers/role_skill_controller_test.exs

```diff
@@ -1,83 +1,42 @@
 defmodule CodeCorps.RoleSkillControllerTest do
-  use CodeCorps.ApiCase
-
-  alias CodeCorps.RoleSkill
-  alias CodeCorps.Repo
-
-  @valid_attrs %{}
-  @invalid_attrs %{}
-
-  defp build_payload, do: %{ "data" => %{"type" => "role-skill", "attributes" => %{}}}
+  use CodeCorps.ApiCase, resource_name: :role_skill
 
   describe "index" do
     test "lists all entries on index", %{conn: conn} do
       path = conn |> role_skill_path(:index)
-      json = conn |> get(path) |> json_response(200)
-      assert json["data"] == []
+      conn = conn |> get(path)
+      assert json_response(conn, 200)["data"] == []
```

Comment:
I would keep this roughly as it is in some of the closed PRs. `conn |> get(path) |> json_response(200)`, then assign the response and assert against that.